### PR TITLE
gh-276: add tests for `glass.shapes`

### PR DESCRIPTION
**Pull request by Saransh-cpp**
Adds tests for `glass.shapes.triaxial_axis_ratio` and `glass.shapes.ellipticity_ryden04` inspired by `skypy`.
Sets up `.coveragrc` (coverage config file) to ignore unrelated files.

Closes: #276

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[run]
+omit = glass/_version.py
```
Comment (paddyroddy) [inline .coveragerc:2]: Can we make coverage stuff a separate PR?

File: glass/shapes.py [hunk 2]
```diff
@@ -31,7 +31,7 @@
 import numpy as np
 
 if TYPE_CHECKING:
-    from numpy.typing import ArrayLike, NDArray
+    from numpy.typing import ArrayLike, NDArray  # pragma: no cover
```
Comment (paddyroddy) [inline glass/shapes.py:34]: I've said this in a separate PR, but I think we should do this https://coverage.readthedocs.io/en/latest/excluding.html#advanced-exclusion

**Review by paddyroddy — CHANGES_REQUESTED** [review]

File: .coveragerc (hunk 1 above)
Comment (Saransh-cpp) [reply .coveragerc:2]: Yes, was about to do that.

File: glass/shapes.py (hunk 2 above)
Comment (Saransh-cpp) [inline glass/shapes.py:34]:
```suggestion
    from numpy.typing import ArrayLike, NDArray
```

**Comment by Saransh-cpp** [comment]
Need to add some comments and docstring.

**Review by Saransh-cpp — COMMENTED** [review]
Hi @paddyroddy, could you please let me know (and possibly link an example for) what kind of docstrings should be added to the tests?

**Comment by paddyroddy** [comment]
> Hi @paddyroddy, could you please let me know (and possibly link an example for) what kind of docstrings should be added to the tests?

If you think it's overkill, feel free to skip. To the uninitiated it just seems like a bunch of random input/outputs, so just a little explanation would be helpful IMO

**Review by paddyroddy — APPROVED** [review]
Just seen the comments, much better. Any reason why the blank line between comment and test?

**Comment by Saransh-cpp** [comment]
> Any reason why the blank line between comment and test?

Really not a fan of the extra blank lines, but again, just following the way it is done in the rest of the existing tests to maintain uniformity😅

**Comment by paddyroddy** [comment]
> Really not a fan of the extra blank lines, but again, just following the way it is done in the rest of the existing tests to maintain uniformity😅

Okay makes sense!
